generators/app/steps/install.js

```diff
@@ -17,8 +19,34 @@ function _onNpmInstallClose(error) {
   }
 }
 
+function _getSelectedDbAdapter() {
```

Comment:
We can make it more easy, without reading configuration file. Each of answers in database section has same name as their appropriate adapters. So you can spawn install just with prefixing it with `sails-`.